src/components/RequiredText/RequiredText.js

```diff
@@ -0,0 +1,29 @@
+import {colors} from '../styles/colors';
+import {typography} from '../styles/typography';
+import styled from 'styled-components';
+
+export const requiredText = styled.div`
```

Comment:
You can change the component name to RequiredText so in SelectInputLabelBox.js and TextInput.js you can import as
```suggestion
import { RequiredText } from [this file]
```
So you don't need the line
```suggestion
export const RequiredText = requiredText;
```
in those two files.
Other than these two, likes pretty good:)